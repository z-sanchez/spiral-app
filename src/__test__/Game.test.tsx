import "@testing-library/jest-dom";

//this solves the jest-vite incompatibility when using import.meta.env
jest.mock("../utils/constants", () => ({
  ENV_VARIABLES: {
    useMockData: true,
  },
}));

describe("New Game Component Functions", () => {
  test("Pick away team", () => {
    expect(3).toBe(3);
  });

  test("Pick home team", () => {
    expect(3).toBe(3);
  });

  test("Switch pick", () => {
    expect(3).toBe(3);
  });
});

describe("New Game Component UI", () => {
  test("Pick is correct", () => {
    expect(3).toBe(3);
  });

  test("Pick is not correct", () => {
    expect(3).toBe(3);
  });

  test("Game is live and game is readonly", () => {
    expect(3).toBe(3);
  });

  test("Using correct NYJ color", () => {
    expect(3).toBe(3);
  });

  test("Using correct IND color", () => {
    expect(3).toBe(3);
  });

  test("Using correct ARI color", () => {
    expect(3).toBe(3);
  });

  test("Show pick scores", () => {
    expect(3).toBe(3);
  });
});
