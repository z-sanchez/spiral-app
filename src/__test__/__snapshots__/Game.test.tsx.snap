// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New Game Component UI Away team picked and highlighted -- Snapshot Test 1`] = `
<div
  className="py-4 flex justify-between"
>
  <div
    className="shadow-[0_0_4px_0_rgba(0,0,0,0.1)] w-[49%] px-2 py-4 rounded-md flex items-center justify-between text-xs"
    data-testid="awayTeam"
    onClick={[Function]}
    style={
      {
        "background": "rgba(168, 85, 247, 0.04)",
        "outline": "1px solid rgba(168, 85, 247, 0.76)",
      }
    }
  >
    <div
      className="flex items-center"
    >
      <p
        style={
          {
            "color": "#000000",
          }
        }
      >
        CIN
         
        <span
          className="font-bold text-[0.85rem]"
        >
          Bengals
        </span>
      </p>
      <p
        className="text-xs text-gray-400 text-center ml-1"
      >
        (
        5
         - 
        6
        )
      </p>
    </div>
  </div>
  <div
    className="shadow-[0_0_4px_0_rgba(0,0,0,0.1)] w-[49%] px-2 py-4 rounded-md flex items-center justify-between text-xs"
    data-testid="homeTeam"
    onClick={[Function]}
    style={{}}
  >
    <div
      className="flex items-center"
    >
      <p
        style={
          {
            "color": "#007487",
          }
        }
      >
        JAX
         
        <span
          className="font-bold text-[0.85rem]"
        >
          Jaguars
        </span>
      </p>
      <p
        className="text-xs text-gray-400 text-center ml-1"
      >
        (
        8
         - 
        3
        )
      </p>
    </div>
  </div>
</div>
`;
